R/gs_spending_bound.R

```diff
@@ -81,39 +81,6 @@
 #' @importFrom stats qnorm
 #'
 #' @export
-#'
```

Comment:
Many refer people to the vignettes? 

Or we should create examples without h1() function

Reply:
If we only show the 1st analysis bound, then we can get rid of `h1`. Otherwise, if we show the 2nd analysis bound, then we need `h1`. I personally feel the 2nd analysis is necessary... for the presentation of a comprehensive story.

I am open to referring people to vignettes in `@examples`, and will defer to @nanxstats for his suggestions.

Reply:
Decision: change the examples in gs_spending_bound as we have in gs_design_ahr.